src/mlpack/methods/logistic_regression/logistic_regression_function.hpp

```diff
@@ -142,6 +142,11 @@ class LogisticRegressionFunction
   double EvaluateWithGradient(const arma::mat& parameters,
                               GradType& gradient) const;
 
+  /**
+   * Evaluate the objective function and gradient of the logistic regression
+   * log-likelihood function simultaneously with the given parameters, for
+   * the given batch size from a given point the in dataset.
+   */
```

Comment:
Looks like there is no description for the constructor as well, do you like to fix that as well? Don't feel obligated. Also, not sure why it's not done here, but usually we provide a description for each parameter, which I think is essential for the documentation processing step. Let me know what you think.

Reply:
@zoq I included the documentation for the constructors and also did minor documentation fixUps.